src/workers/parser/getSymbols.js

```diff
@@ -199,7 +200,12 @@ function extractSymbol(path: SimplePath, symbols) {
       return;
     }
 
-    if (t.isProperty(path.parent)) {
+    if (
+      path.parent &&
+      t.isProperty(path.parent) &&
+      !t.isAssignmentPattern(path.getAncestor(3)) &&
+      !(t.isCallExpression(path.getAncestor(3)) && keyIsValue(path.parent))
```

Comment:
Line 206 and 207, can you please explain what they are doing?

Reply:
basically this case says that if the grandparent in the AST is either a Assignment pattern, or a Call expression with one variable in it.... like for example dispatch({ a })